isthmus/src/test/java/io/substrait/isthmus/utils/SetUtils.java

```diff
@@ -21,9 +21,9 @@ public static String getSetQuery(Set.SetOp op, boolean multi) {
     String opString =
         switch (op) {
           case MINUS_PRIMARY -> "EXCEPT";
-          case MINUS_MULTISET -> "EXCEPT ALL";
-          case INTERSECTION_PRIMARY -> "INTERSECT";
-          case INTERSECTION_MULTISET -> "INTERSECT ALL";
```

Comment:
This only appears to be used in tests. I think it would be better to not map the operations that don't correspond to SQL operations as we don't have to worry about compatibility.

It looks like this is used with `setTestConfig`, below which is what the tests use to generated the Set Operators that are being test. We can likely just filter out the unsupported SetOps there are as well.